rll-examples/rightscale-mirrors.sh

```diff
@@ -1,6 +1,8 @@
 #! /bin/bash -e
 
-# RightScale provides mirrors of some OS distributions. Snapshots of these mirrors
+# ---
+# RightScript Name: RL10 Linux RightScale Mirrors
+# Description: RightScale provides mirrors of some OS distributions. Snapshots of these mirrors
 # are taken daily so that any the mirrors can be "frozen" to any given day. These
```

Comment:
In YAML, I think that subsequent lines to continue a string need to be indented, so this should be something like:

```bash
# Description: RightScale provides mirrors of some OS distributions. Snapshots of these mirrors
#   are taken daily so that any the mirrors can be "frozen" to any given day. These
```